authentication/authentication-common/src/test/java/io/mosip/authentication/common/service/impl/IdServiceImplTest.java

```diff
@@ -290,27 +285,27 @@ public void getTokenTest() {
 	@Test
 	public void updateVidStatusTest() throws IdAuthenticationBusinessException {
 		String vid = "213131";
-		IdentityEntity entity = getEntity();
-		entity.setTransactionLimit(2);
+		Optional<IdentityEntity> entityOpt = Optional.of(getEntity());
```

Comment:
Split each use case as separate test method, giving proper name to the test method as per use case. Add assertion about the transaction limit and delete call..

Reply:
Added assertion as well as created separate test methods